handler.py

```diff
@@ -35,7 +42,11 @@ def report_cost(event, context):
     paginator = iam.get_paginator('list_account_aliases')
     account_name = '[NOT FOUND]'
     for aliases in paginator.paginate(PaginationConfig={'MaxItems': 1}):
-        account_name = aliases['AccountAliases'][0]
+        if ('AccountAliases' in aliases):
+            try:
+                account_name = aliases['AccountAliases'][0]
+            except IndexError:
+                pass
```

Comment:
I think an extra step on the `if` clause should make it so you don't need the try/except:

```suggestion
        if 'AccountAliases' in aliases and len(aliases['AccountAliases']) > 0:
            account_name = aliases['AccountAliases'][0]
```